client/app/components/ServicesTemplate/index.js

```diff
@@ -13,13 +13,15 @@ const ServicesTemplate = () => {
     <Card
       key={id}
       cardSize="is-6"
-      {...{ title,  content, url, img }}
+      img={<img role="presentation" src={require(`./images/${img}.svg`)} style={{ fill: '#000066'}} />}
+      {...{ title,  content, url}}
```

Comment:
Same as below. Generally, if you're copy pasting code, it means you can do some form of abstraction

Reply:
i will put this in a different story and prioritize after demo.